htdocs/core/class/commonobject.class.php

```diff
@@ -3286,73 +3287,78 @@ function printObjectLines($action, $seller, $buyer, $selected=0, $dateSelector=0
 		$usemargins=0;
 		if (! empty($conf->margin->enabled) && ! empty($this->element) && in_array($this->element,array('facture','propal','commande'))) $usemargins=1;
 
-		print '<tr class="liste_titre nodrag nodrop">';
-
-		if (! empty($conf->global->MAIN_VIEW_LINE_NUMBER)) print '<td class="linecolnum" align="center" width="5">&nbsp;</td>';
+		$parameters = array();
+		$reshook = $hookmanager->executeHooks('printObjectHeads', $parameters, $this, $action); // Note that $action and $object may have been modified by some hooks
+		if (empty($hookmanager->resPrint))
```

Comment:
Hook must be "replace" hook. It means we must test the value of $reshook to know if we execute standard code or not.

$reshook = $hookmanager->executeHooks('printObjectHeads', $parameters, $this, $action);
print $hookmanager->resPrint;
If (empty($reshook)) 
{
    ...
}
else
{

}